README.md

```diff
@@ -51,6 +51,56 @@ async function main() {
 }
 
 ```
+### reviewPullRequest(options)
+
+The `reviewPullRequest()` method creates a code suggestion review on a GitHub Pull request with the files given as input.
+From these files, calculate the hunk diff and make all the multiline code suggestion review comments on a given pull request with these hunks given
+that they are in scope of the pull request. Outof scope suggestions are not made.
+
+In-scope suggestions are specifically: suggestions whose file is in-scope of the pull request,
+and suggestions whose diff hunk is a subset of the pull request's files hunks.
+ 
+ If a file is too large to load in the review, it is skipped in the suggestion phase.
+
+ If the program terminates without exception, a timeline comment will be made with all errors or suggestions that could not be made.
+
+#### Syntax
+`reviewPullRequest(octokit, rawChanges, config [, logger])`
+
+#### Parameters
+#### `octokit`
+*octokit* <br>
+**Required.** An authenticated [octokit](https://github.com/octokit/rest.js/) instance.
+
+#### `rawChanges`
```

Comment:
The "raw" part doesn't mean much to me. Is there any way we can consolidate the changes types between `createPullRequest` and `reviewPullRequest`?